cmds/exp/uefiboot/main.go

```diff
@@ -37,8 +37,6 @@ var (
 	execute     = flag.Bool("e", true, "If true, kexec. If not true, just load kernel.")
 )
 
-var v = func(string, ...interface{}) {}
```

Comment:
is this unused because someone made a mistake and did not use it?

Reply:
It looks to me like that, probably someone left this during debugging.